### ISSUE TITLE
Support App Platform alerts via slack

### ISSUE THREAD

**Issue by hstaudacher** [digitalocean/terraform-provider-digitalocean#802]
### Is your feature request related to a problem? Please describe.
When configuring an alert within the digitalocean_app spec only an email alert will be configured. It would be nice if one can choose between email/slack as within the do UI.

**Comment by andrewsomething** [comment]
Thanks for opening an issue to track this. As mentioned in [the PR adding initial alert support](https://github.com/digitalocean/terraform-provider-digitalocean/pull/797), destinations can not be configured in the [app spec itself](https://docs.digitalocean.com/products/app-platform/references/app-specification-reference/). You need to call a separate endpoint to add Slack hooks or other emails:

https://docs.digitalocean.com/reference/api/api-reference/#operation/assign_alert_destinations

This might end up needing to be a separate app alert destinations resource.

**Comment by hstaudacher** [comment]
I wasn't aware of #797. Thx for pointing me to it. The problem I see right now is that the endpoint you are mentioning requires an alert id which is not exposed when creating the app in terraform. So, I guess would have to [list all alerts](https://docs.digitalocean.com/reference/api/api-reference/#operation/list_alerts) for all apps and update the destinations. Is this what you would a suggest? If not can you point me to the right direction? Thx in advance.

**Comment by andrewsomething** [comment]
Yes. `doctl`  also has commands for using both of those if the CLI makes it easier for you.

**Comment by ricardo01010** [comment]
Hello, are you planning to include this in the module? That is, the integration of slack is native.

**Comment by voxivoid** [comment]
it would be great to support this since it is a very common use case for the alerts

**Comment by hstaudacher** [comment]
Any progress on this?

**Comment by mwyld** [comment]
This would be a great feature :)
